templates/.reek

```diff
@@ -1,6 +1,8 @@
 ---
 Attribute:
   enabled: false
+NilCheck:
+  enabled: false
```

Comment:
Hmmmm... What's the reason for this?

Reply:
It prevents usage of the safe navigation operator (`&.`).